src/System.Net.Security/tests/FunctionalTests/SslStreamNegotiatedCipherSuiteTest.cs

```diff
@@ -0,0 +1,357 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Net.Test.Common;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    using Configuration = System.Net.Test.Common.Configuration;
+
+    public class NegotiatedCipherSuiteTest
+    {
+        private static bool IsKnownPlatformSupportingTls13 => PlatformDetection.IsUbuntu1810OrHigher;
+        private static bool Tls13Supported { get; set; } = ProtocolsSupported(SslProtocols.Tls13);
+
+        private static HashSet<TlsCipherSuite> _tls13CipherSuiteLookup = new HashSet<TlsCipherSuite>(GetTls13CipherSuites());
+        private static HashSet<TlsCipherSuite> _tls12CipherSuiteLookup = new HashSet<TlsCipherSuite>(GetTls12CipherSuites());
+        private static HashSet<TlsCipherSuite> _tls10And11CipherSuiteLookup = new HashSet<TlsCipherSuite>(GetTls10And11CipherSuites());
```

Comment:
Missing `s` for static fields.

Reply:
missed one variable, already renamed in my next PR